extensions/control-plane/api/management-api/policy-definition-api/src/main/java/org/eclipse/edc/connector/controlplane/api/management/policy/v31alpha/PolicyDefinitionApiV31AlphaController.java

```diff
@@ -101,4 +106,26 @@ public JsonObject validatePolicyDefinitionV3(@PathParam("id") String id) {
         return transformerRegistry.transform(validationResult, JsonObject.class)
                 .orElseThrow(f -> new EdcException("Error creating response body: " + f.getFailureDetail()));
     }
+
+    @POST
+    @Path("{id}/evaluationPlan")
```

Comment:
we never used camelCase for path items, mostly we kept everything all underscored (`evaluationplan`), sometimes kebab case (`evaluation-plan`). Generally I prefer the latter the most, but the first is more used in the codebase